### PR TITLE
No error message shows when using wrong/expired cached credentials #2…

### LINKED ISSUE #2181: No error message shows when using wrong/expired cached credentials🐛
When clicking "Switch User" to log in with cached credentials, no error message is shown if the credentials are no longer valid. This happends when you try to log in using a system user from Azure that is no longer valid. 

The expected behavior is to get the same error message you get when you try to log in manually:
image

### PR DESCRIPTION
**Pull request by robertbasti**
[//]: # (Request Template Witsml Explorer)

[//]: # (Thank you for contributing to WITSML Explorer! Before submitting this PR, please fill in the following template.)

## Fixes
 This pull request fixes https://github.com/equinor/witsml-explorer/issues/2181

## Description
-  display error message is shown when cached credentials are no longer valid
-  replace api method to POST

[//]: # (Please include a written summary of the changes and which issue is fixed. List any dependencies that are required for this change._)

## Type of change

[//]: # (Mark any of the types of change that apply.)

* [x] Bugfix
* [ ] New feature (non-breaking change which adds functionality)
* [x] Enhancement of existing functionality
* [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
* [ ] This change requires a documentation update

## Impacted Areas in Application

[//]: # (List general components of the application that this PR will affect)

* [x] Frontend
* [x] API
* [ ] WITSML
* [ ] Other (please describe)

## Checklist:

[//]: # (Please tick all the boxes or remove the ones that aren't needed and explain why)

*Communication*
* [ ] I have made corresponding changes to the documentation
* [ ] PR affects application security

*Code quality*
* [x] I have self-reviewed my code
* [x] No new warnings are generated

*Test coverage*
* [] New code is covered by passing tests

## Further comments

[//]: # (If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...)

### CODE REVIEW

File: Src/WitsmlExplorer.Api/Models/ConnectionInformation.cs [hunk 1]
```diff
@@ -0,0 +1,19 @@
+using System;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+
+namespace WitsmlExplorer.Api.Models
+{
+    public class ConnectionInformation
+    {
+        [JsonPropertyName("url")]
+        public Uri Url { get; init; }
```
Comment (eliasbruvik) [inline Src/WitsmlExplorer.Api/Models/ConnectionInformation.cs]: Can you change this field to ServerUrl just to make it even more clear that this is connected to witsml server urls?

File: Src/WitsmlExplorer.Frontend/services/authorizationService.ts [hunk 2]
```diff
@@ -29,6 +29,11 @@ export interface AuthorizationState {
   status: AuthorizationStatus;
 }
 
+export interface ConnectionInformation {
+  url: string;
```
Comment (eliasbruvik) [inline Src/WitsmlExplorer.Frontend/services/authorizationService.ts]: Can you change this field to serverUrl just to make it even more clear that this is connected to witsml server urls?

**Review by eliasbruvik — CHANGES_REQUESTED** [review]

File: Src/WitsmlExplorer.Frontend/services/authorizationService.ts (hunk 2 above)
Comment (robertbasti) [reply Src/WitsmlExplorer.Frontend/services/authorizationService.ts]: done

File: Src/WitsmlExplorer.Api/Models/ConnectionInformation.cs (hunk 1 above)
Comment (robertbasti) [reply Src/WitsmlExplorer.Api/Models/ConnectionInformation.cs]: done

**Review by eliasbruvik — APPROVED** [review]
- [x] Read issue and PR
- [x] Pulled branch and manually tested
- [x] Verify that PR resolves issue
- [x] Reviewed the code

**Pull request merged**